src/store/getters.ts

```diff
@@ -69,7 +69,7 @@ export const isMobile = (state: StateIF): boolean => {
 
 /** True if user is authenticated, else False. */
 export const isAuthenticated = (rootState: RootStateIF): boolean => {
-  return Boolean(rootState.auth && rootState.auth.token)
+  return Boolean(rootState.auth?.token)
```

Comment:
Nice.

I would add a ? to rootState as well, in case it's null or undefined.

Reply:
But I see we don't do that for "state" in the other getters, so you don't have to do this.